zooniverse/front-end-monorepo
# Add app-content-pages dev server with devcert

## Package
- app-content-pages

## Why and Context
- when working on app-content-pages locally, I can't login to Panoptes
- this PR is similar to #2310
- creates a local certificate authority to generate self-signed SSL certificates (more info on how this works: https://github.com/davewasmer/devcert#how-it-works)
- adding this will enable us to use https://local.zooniverse.org:3000/about/team or https://localhost.zooniverse.org:3000/about/team with working auth with Panoptes

## Describe your changes
- adds dev server with devcert, dev server ran with `yarn dev`
- update README accordingly ^

## How to Review
_Helpful explanations that will make your reviewer happy:_
- in app-content-pages, run `yarn dev`
- go to https://local.zooniverse.org:3000/about/team or https://localhost.zooniverse.org:3000/about/team
- try authentication with a staging user, you should be able to authenticate

# Checklist
_PR Creator - Please cater the checklist to fit the review needed for your code changes._
_PR Reviewer - Use the checklist during your review. Each point should be checkmarked or discussed before PR approval._

## General

## New Feature

## Refactoring
